### DISCUSSION TITLE
CS8981

### DISCUSSION THREAD

**Discussion by The-Futurist** [dotnet/csharplang#6822] [Q&A] [answered]
Was this change ever part of a discussion here? I can't find any obvious trace of this here.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/warning-waves#cs8981---the-type-name-only-contains-lower-cased-ascii-characters

**Comment by Joe4evr** [comment, marked as answer]
This was discussed in #4460.

Specifically, this was already pointed out to you in that very thread:
> Related to this topic is the discussion of warning users about the user of all-lower-case type names:
> 
> * dotnet/roslyn#56653
> * https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-10-13.md#warning-on-lowercase-type-names

**Reply by The-Futurist** [reply]
> This was discussed in #4460.
> 
> Specifically, this was already pointed out to you in that very thread:
> 
> > Related to this topic is the discussion of warning users about the user of all-lower-case type names:
> > 
> > * [Compiler or analyzer should warn for lower-cased type names roslyn#56653](https://github.com/dotnet/roslyn/issues/56653)
> > * https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-10-13.md#warning-on-lowercase-type-names

Thanks, I had a feeling this had been touched upon in the past, much appreciated.
